development/ts-migration-dashboard/common/build-module-partitions.ts

```diff
@@ -78,11 +98,21 @@ export default async function buildModulePartitions(): Promise<
     await Promise.all(
       ENTRYPOINT_PATTERNS.map((entrypointPattern) => {
         return fg(
-          path.resolve(ROOT_DIRECTORY_PATH, `${entrypointPattern}.{js,ts,tsx}`),
+          path.resolve(
+            ROOT_DIRECTORY_PATH,
+            `${entrypointPattern}.{js,jsx,ts,tsx}`,
+          ),
         );
       }),
     )
-  ).flat();
+  )
+    .flat()
+    .filter((filePath) => {
+      return !/^(?:\.storybook|node_modules)\//u.test(
```

Comment:
We use the `madge` package to locate `import`s across files so we are able to locate all of the JavaScript and TypeScript files in this repo that are not dead code. We do this by starting with a set of entrypoints. These entrypoints are determined by a set of globals which we then expand into full file paths. After performing the expansion step, the resulting list of entrypoints mistakenly included Storybook files and files in `node_modules`. This change fixes that.